unittests/savanna_misc_tests.cpp

```diff
@@ -585,12 +584,107 @@ BOOST_FIXTURE_TEST_CASE(validate_qc_requiring_finalizer_policies, savanna_cluste
    set_partition({0});                                  // partition A so it doesn't receive blocks on `open()`
    A.open_from_snapshot(b6_snapshot);
 
-#if 0  // uncomment when issue #694 is fixed.
    A.push_block(b7);                                    // when pushing b7, if we try to access any block state
    A.push_block(b8);                                    // before b6, we will fail with a `verify_qc_claim`
    A.push_block(b9);                                    // exception, which is what will happens until issue
                                                         // #694 is addressed.
-#endif
+
+} FC_LOG_AND_RETHROW()
+
+
+
+// ---------------------------------------------------------------------------------------------------
+// For issue #694, we need to change the finality core of the `block_header_state`, but we want to
+// insure that this doesn't create a consensus incompatibility with spring 1.0, so the blocks created
+// with newer versions remain compatible (and linkable) by spring 1.0.
```

Comment:
`insure` -> `ensure`
`spring 1.0` -> `Spring 1.0.0` (in two places)

Reply:
https://github.com/AntelopeIO/spring/pull/719/commits/d272c2517ef206052e74fb9bec037c073e6d5be7